api/v2/api.go

```diff
@@ -238,6 +249,28 @@ func (api *API) getAlertsHandler(params alert_ops.GetAlertsParams) middleware.Re
 			continue
 		}
 
+		// TODO: Add a timestamp now()+X, after which the Mutes() will
+		// be refreshed?
```

Comment:
I think we should use a different strategy, see my earlier comment. But let's not worry about the performance right now.